[Graph] Fix various a11y issues

flash1293 opened the pull request:
## Summary

Fixes https://github.com/elastic/kibana/issues/48488

This PR addresses some a11y issues in Graph:

* Empty listing
  * [x] No h1 heading - changed the prompt to create a new Graph from `h2` to `h1`
* Filled listing
  * [ ] Empty table header (has to be fixed i table list view) - not actually an error - there is no text in the `th` itself, but the checkbox in there has a label. If this is a problem, it has to be fixed in eui.
* Workspace
  * [x] No heading structure (only on filled workspace, on empty workspace guidance panel takes over this role) - added a screenreader visible h1 on filled workspace
* Other a11y problems that can be fixed in the same batch
  * [x] Empty buttons in side bar (no accessible label) - added tooltip as `aria-label` as well
  * [x] Low contrast issue in the side bar - tinted the color a little

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/)
* Commit: 7f8244be6cf294b6d88fe598c6992b92e4f8a208

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/","number":"18329","commit":"7f8244be6cf294b6d88fe598c6992b92e4f8a208"}]}
PIPELINE-->

flash1293 commented:
Jenkins, test this. Not sure whether unrelated or not.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/)
* Commit: 35679b78c2a686339ede987cdcf035f9a3a1ea0f

### History
* :broken_heart: [Build #18329](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/) failed 7f8244be6cf294b6d88fe598c6992b92e4f8a208

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/","number":"18329","commit":"7f8244be6cf294b6d88fe598c6992b92e4f8a208"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/","number":"18388","commit":"35679b78c2a686339ede987cdcf035f9a3a1ea0f"}]}
PIPELINE-->

elasticmachine commented:
Pinging @elastic/kibana-app (Team:KibanaApp)

myasonik commented:
Saw a few more low hanging fruit for a11y fixes here. If you have a chance, could you add these fixes in as well? (If anything's not clear, feel free to reach out! Happy to zoom!) 

- Can you add `aria-descrbiedby={idOfTheH1}` to the `main` on the graph empty page? (`EuiPageBody` renders as a `main` and you _should_ be able to pass that prop in...) 
- When going through the steps to add a graph, can the contents of the page be put inside a `main` tag? (`#graphBasic` might be a good option to turn into a `main` which I'm guessing you'll have control over given its id and should also solve the problem for the workspace in general...) (And, same as item above, can you add `aria-describedby` the same way to it as well?)
	- Should also work for the workspace's `h1`
- Can `ul.gphGuidancePanel__list` be changed to an `<ol>`? Also on this element, can you add `aria-describedby={idOfTheH1}`?
- For each `li.gphGuidancePanel__item`, could you add `aria-current="step"` to the current one that should be dealt with?
- Adding `display: contents` onto buttons (i.e., `.gphGuidancePanel__item button`) unfortunately ruins their semantics and so several of their interactions (like focus stats and tabbing). Toggling it off in the inspector I don't see any changes without the property so I'm hoping we can just remove it?
- You kill pointer events on buttons for disabled items but I'm still able to activate the button using a screen reader. Could you add `disabled={true}` to any buttons that are inside disabled list items?

flash1293 commented:
Hi @myasonik ,

thanks for this!

I went through your points and I think I addressed all of them.

> Adding display: contents onto buttons (i.e., .gphGuidancePanel__item button) unfortunately ruins their semantics and so several of their interactions (like focus stats and tabbing). Toggling it off in the inspector I don't see any changes without the property so I'm hoping we can just remove it?

This was originally added to have the button text wrap like ordinary text, but with the current labels it's not necessary anymore because the clickable texts are pretty short. I removed it for now.

Can you add aria-descrbiedby={idOfTheH1} to the main on the graph empty page? (EuiPageBody renders as a main and you should be able to pass that prop in...)
When going through the steps to add a graph, can the contents of the page be put inside a main tag? (#graphBasic might be a good option to turn into a main which I'm guessing you'll have control over given its id and should also solve the problem for the workspace in general...) (And, same as item above, can you add aria-describedby the same way to it as well?)
Should also work for the workspace's h1
Can ul.gphGuidancePanel__list be changed to an <ol>? Also on this element, can you add aria-describedby={idOfTheH1}?
For each li.gphGuidancePanel__item, could you add aria-current="step" to the current one that should be dealt with?
Adding display: contents onto buttons (i.e., .gphGuidancePanel__item button) unfortunately ruins their semantics and so several of their interactions (like focus stats and tabbing). Toggling it off in the inspector I don't see any changes without the property so I'm hoping we can just remove it?
You kill pointer events on buttons for disabled items but I'm still able to activate the button using a screen reader. Could you add disabled={true} to any buttons that are inside disabled list items?

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/)
* Commit: 9f6853f40048d1a7b9b06fefc97d599c317ca6f0

### History
* :green_heart: [Build #18388](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/) succeeded 35679b78c2a686339ede987cdcf035f9a3a1ea0f
* :broken_heart: [Build #18329](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/) failed 7f8244be6cf294b6d88fe598c6992b92e4f8a208

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/","number":"18329","commit":"7f8244be6cf294b6d88fe598c6992b92e4f8a208"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/","number":"18388","commit":"35679b78c2a686339ede987cdcf035f9a3a1ea0f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/","number":"18628","commit":"9f6853f40048d1a7b9b06fefc97d599c317ca6f0"}]}
PIPELINE-->

lizozom commented on src/plugins/kibana_react/public/table_list_view/table_list_view.tsx:
```diff
@@ -67,6 +67,8 @@ export interface TableListViewProps {
   tableListTitle: string;
   toastNotifications: ToastsStart;
   uiSettings: IUiSettingsClient;
+  ariaDescribedby?: string;
```
`TableListView` already takes in quite a few textual inputs (`entityName`, `entityNamePlural`, `tableListTitle`).
Could those be used for this?

If the fact they are translated is an issue, I would still pass in a single input, instead of the two added here (`ariaDescribedby`, `headingId`)?.

Also, `TableListView` is used in only two other places. Would you mind adding this new attribute (if eventually needed) there as well?

flash1293 replied:
The reason I introduced new properties is that in some cases the same id has to be used outside of the component. I'm fine with just using a single property - it takes away some flexibility in using the component, but for the current usage it's not relevant.

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/)
* Commit: 2c87d99f5312b39db9b21a1908a8d57d7eac5a4d

### History
* :green_heart: [Build #18628](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/) succeeded 9f6853f40048d1a7b9b06fefc97d599c317ca6f0
* :green_heart: [Build #18388](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/) succeeded 35679b78c2a686339ede987cdcf035f9a3a1ea0f
* :broken_heart: [Build #18329](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/) failed 7f8244be6cf294b6d88fe598c6992b92e4f8a208

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/","number":"18329","commit":"7f8244be6cf294b6d88fe598c6992b92e4f8a208"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/","number":"18388","commit":"35679b78c2a686339ede987cdcf035f9a3a1ea0f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/","number":"18628","commit":"9f6853f40048d1a7b9b06fefc97d599c317ca6f0"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/","number":"18677","commit":"2c87d99f5312b39db9b21a1908a8d57d7eac5a4d"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/)
* Commit: c46081943b5bd0dd6654048f1ceaa3b5d4896dac

### History
* :broken_heart: [Build #18677](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/) failed 2c87d99f5312b39db9b21a1908a8d57d7eac5a4d
* :green_heart: [Build #18628](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/) succeeded 9f6853f40048d1a7b9b06fefc97d599c317ca6f0
* :green_heart: [Build #18388](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/) succeeded 35679b78c2a686339ede987cdcf035f9a3a1ea0f
* :broken_heart: [Build #18329](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/) failed 7f8244be6cf294b6d88fe598c6992b92e4f8a208

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/","number":"18329","commit":"7f8244be6cf294b6d88fe598c6992b92e4f8a208"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/","number":"18388","commit":"35679b78c2a686339ede987cdcf035f9a3a1ea0f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/","number":"18628","commit":"9f6853f40048d1a7b9b06fefc97d599c317ca6f0"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/","number":"18677","commit":"2c87d99f5312b39db9b21a1908a8d57d7eac5a4d"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/","number":"18704","commit":"c46081943b5bd0dd6654048f1ceaa3b5d4896dac"}]}
PIPELINE-->

myasonik commented on src/plugins/kibana_react/public/table_list_view/table_list_view.tsx:
```diff
@@ -498,7 +503,9 @@ class TableListView extends React.Component<TableListViewProps, TableListViewSta
         restrictWidth
       >
-        <EuiPageBody>{this.renderPageContent()}</EuiPageBody>
+        <EuiPageBody aria-describedby={this.props.headingId}>
```
```suggestion
        <EuiPageBody aria-labelledby={this.props.headingId}>
```

myasonik commented on x-pack/legacy/plugins/graph/public/angular/templates/index.html:
```diff
@@ -1,4 +1,4 @@
-<div id="graphBasic" ng-controller="graphuiPlugin">
+<main id="graphBasic" ng-controller="graphuiPlugin" aria-describedby="graphHeading">
```
```suggestion
<main id="graphBasic" ng-controller="graphuiPlugin" aria-labelledby="graphHeading">
```

myasonik commented on x-pack/legacy/plugins/graph/public/components/guidance_panel/guidance_panel.tsx:
```diff
@@ -96,15 +97,15 @@ function GuidancePanelComponent(props: GuidancePanelProps) {
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
-          <ul className="gphGuidancePanel__list">
+          <ol className="gphGuidancePanel__list" aria-describedby="graphHeading">
```
```suggestion
          <ol className="gphGuidancePanel__list" aria-labelledby="graphHeading">
```

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/)
* Commit: 17bff7b25474a9827c0419f7fe182a8c5ca3e071

### History
* :green_heart: [Build #18704](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/) succeeded c46081943b5bd0dd6654048f1ceaa3b5d4896dac
* :broken_heart: [Build #18677](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/) failed 2c87d99f5312b39db9b21a1908a8d57d7eac5a4d
* :green_heart: [Build #18628](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/) succeeded 9f6853f40048d1a7b9b06fefc97d599c317ca6f0
* :green_heart: [Build #18388](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/) succeeded 35679b78c2a686339ede987cdcf035f9a3a1ea0f
* :broken_heart: [Build #18329](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/) failed 7f8244be6cf294b6d88fe598c6992b92e4f8a208

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18329/","number":"18329","commit":"7f8244be6cf294b6d88fe598c6992b92e4f8a208"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/","number":"18388","commit":"35679b78c2a686339ede987cdcf035f9a3a1ea0f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/","number":"18628","commit":"9f6853f40048d1a7b9b06fefc97d599c317ca6f0"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/","number":"18677","commit":"2c87d99f5312b39db9b21a1908a8d57d7eac5a4d"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/","number":"18704","commit":"c46081943b5bd0dd6654048f1ceaa3b5d4896dac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/","number":"19212","commit":"17bff7b25474a9827c0419f7fe182a8c5ca3e071"}]}
PIPELINE-->

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19217/)
* Commit: 3cd3ed51f7bdc006554b2d52aded8b74d77b053e

### History
* :broken_heart: [Build #19212](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/) failed 17bff7b25474a9827c0419f7fe182a8c5ca3e071
* :green_heart: [Build #18704](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/) succeeded c46081943b5bd0dd6654048f1ceaa3b5d4896dac
* :broken_heart: [Build #18677](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/) failed 2c87d99f5312b39db9b21a1908a8d57d7eac5a4d
* :green_heart: [Build #18628](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/) succeeded 9f6853f40048d1a7b9b06fefc97d599c317ca6f0
* :green_heart: [Build #18388](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/) succeeded 35679b78c2a686339ede987cdcf035f9a3a1ea0f

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18388/","number":"18388","commit":"35679b78c2a686339ede987cdcf035f9a3a1ea0f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/","number":"18628","commit":"9f6853f40048d1a7b9b06fefc97d599c317ca6f0"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/","number":"18677","commit":"2c87d99f5312b39db9b21a1908a8d57d7eac5a4d"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/","number":"18704","commit":"c46081943b5bd0dd6654048f1ceaa3b5d4896dac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/","number":"19212","commit":"17bff7b25474a9827c0419f7fe182a8c5ca3e071"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19217/","number":"19217","commit":"3cd3ed51f7bdc006554b2d52aded8b74d77b053e"}]}
PIPELINE-->

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19418/)
* Commit: 0554990af40703caeb29607d1993824a67b1e719

### History
* :broken_heart: [Build #19217](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19217/) failed 3cd3ed51f7bdc006554b2d52aded8b74d77b053e
* :broken_heart: [Build #19212](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/) failed 17bff7b25474a9827c0419f7fe182a8c5ca3e071
* :green_heart: [Build #18704](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/) succeeded c46081943b5bd0dd6654048f1ceaa3b5d4896dac
* :broken_heart: [Build #18677](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/) failed 2c87d99f5312b39db9b21a1908a8d57d7eac5a4d
* :green_heart: [Build #18628](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/) succeeded 9f6853f40048d1a7b9b06fefc97d599c317ca6f0

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18628/","number":"18628","commit":"9f6853f40048d1a7b9b06fefc97d599c317ca6f0"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/","number":"18677","commit":"2c87d99f5312b39db9b21a1908a8d57d7eac5a4d"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/","number":"18704","commit":"c46081943b5bd0dd6654048f1ceaa3b5d4896dac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/","number":"19212","commit":"17bff7b25474a9827c0419f7fe182a8c5ca3e071"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19217/","number":"19217","commit":"3cd3ed51f7bdc006554b2d52aded8b74d77b053e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19418/","number":"19418","commit":"0554990af40703caeb29607d1993824a67b1e719"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19437/)
* Commit: 1cc7a0d89abf9e4fd646d3232452a0ea5518f351

### History
* :broken_heart: [Build #19418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19418/) failed 0554990af40703caeb29607d1993824a67b1e719
* :broken_heart: [Build #19217](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19217/) failed 3cd3ed51f7bdc006554b2d52aded8b74d77b053e
* :broken_heart: [Build #19212](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/) failed 17bff7b25474a9827c0419f7fe182a8c5ca3e071
* :green_heart: [Build #18704](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/) succeeded c46081943b5bd0dd6654048f1ceaa3b5d4896dac
* :broken_heart: [Build #18677](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/) failed 2c87d99f5312b39db9b21a1908a8d57d7eac5a4d

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18677/","number":"18677","commit":"2c87d99f5312b39db9b21a1908a8d57d7eac5a4d"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/","number":"18704","commit":"c46081943b5bd0dd6654048f1ceaa3b5d4896dac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/","number":"19212","commit":"17bff7b25474a9827c0419f7fe182a8c5ca3e071"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19217/","number":"19217","commit":"3cd3ed51f7bdc006554b2d52aded8b74d77b053e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19418/","number":"19418","commit":"0554990af40703caeb29607d1993824a67b1e719"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19437/","number":"19437","commit":"1cc7a0d89abf9e4fd646d3232452a0ea5518f351"}]}
PIPELINE-->

flash1293 commented:
@elasticmachine merge upstream

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19653/)
* Commit: e6c1c89607da92e708844ad6b63b28f7fcde87e5

### History
* :green_heart: [Build #19437](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19437/) succeeded 1cc7a0d89abf9e4fd646d3232452a0ea5518f351
* :broken_heart: [Build #19418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19418/) failed 0554990af40703caeb29607d1993824a67b1e719
* :broken_heart: [Build #19217](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19217/) failed 3cd3ed51f7bdc006554b2d52aded8b74d77b053e
* :broken_heart: [Build #19212](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/) failed 17bff7b25474a9827c0419f7fe182a8c5ca3e071
* :green_heart: [Build #18704](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/) succeeded c46081943b5bd0dd6654048f1ceaa3b5d4896dac

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/18704/","number":"18704","commit":"c46081943b5bd0dd6654048f1ceaa3b5d4896dac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19212/","number":"19212","commit":"17bff7b25474a9827c0419f7fe182a8c5ca3e071"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19217/","number":"19217","commit":"3cd3ed51f7bdc006554b2d52aded8b74d77b053e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19418/","number":"19418","commit":"0554990af40703caeb29607d1993824a67b1e719"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19437/","number":"19437","commit":"1cc7a0d89abf9e4fd646d3232452a0ea5518f351"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/19653/","number":"19653","commit":"e6c1c89607da92e708844ad6b63b28f7fcde87e5"}]}
PIPELINE-->
